hub/herald/session.py

```diff
@@ -1106,6 +1106,9 @@ async def send_notification(self, method, args=()) -> bool:
             self.abort()
             return False
 
+    def send_notification(self, method, args=()):
+        self._task_group.add(self._send_notification(method, args))
+
     async def send_notifications(self, notifications) -> bool:
```

Comment:
Just for consistency with `_send_notification/send_notification`, this should be named `_send_notifications` as it's the non-task-creating version.